src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
Toolbar height changes 3 times depending on the width of the page, but it is not cosistent behavior. I've different results on windows chrome and on linux chrome and on linux firefox. This is not related to mdMedia service and feels more like a bug actually.